samples/servicecontrol/adapter-sqlserver/sample.md

```diff
@@ -0,0 +1,108 @@
+ 1. [Install ServicePulse](https://docs.particular.net/servicepulse/installation)
+
+NOTE: In order to use a different SQL Server instance than local SQL Server Express, make sure the connection strings are updated in all places.
+
+## Running the project
+
+ 1. Start the following projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option).
+ 1. Open ServicePulse and select the Endpoints Overview view. The `Samples.ServiceControl.SqlServerTransportAdapter.Shipping` endpoint should be visible in the Active Endpoints tab as it has the Heartbeats plugin installed
+ 1. Go to the Sales console and press `o` to create an order.
+ 1. Notice the Shipping endpoint receives the `OrderAccepted` event from Sales and publishes an `OrderShipped` event.
+ 1. Notice the Sales endpoint logs that it processed the `OrderShipped` event. 
+ 1. Go to the Sales console and press `f` to simulate a database failure.
+ 1. Press `o` to create another order. Notice the `OrderShipped` event fails processing in Sales and is moved to the error queue
+ 1. Press `f` again to disable database failure simulation in the Sales console.
+ 1. Go to the Shipping console and press `f` to simulate database failure.
+ 1. Go back to the Sales console and press `o` to create yet another order. Notice the `OrderAccepted` event fails in Shipping and is moved to the error queue.
+ 1. Press `f` again to disable database failure simulation in the Shipping console.
+ 1. Open ServicePulse and select the Failed Messages view.
+ 1. Notice one failed message group `System.Exception: Database.Store()` with two messages. Open the group.
+ 1. One of the messages is `OrderAccepted` which failed in `Shipping`, the other is `OrderShipped` which failed in `Sales`.
+ 1. Press the "Retry all" button.
+ 1. Go to the Shipping console and verify that the `OrderAccepted` event has been successfully processed.
+ 1. Go to the Sales console and verify that both `OrderShipped` events have been successfully processed.
+ 1. Shut down the Shipping endpoint.
+ 1. Open ServicePulse and notice a red label next to the heart icon. Click on the that icon to open the Endpoints Overview. Notice that `Samples.ServiceControl.SqlServerTransportAdapter.Shipping` is now displayed in the Inactive Endpoints tab.
+
+
+## Code walk-through 
+
+The solution consists of four projects.
+
+### Shared
+
+The Shared project contains the message definitions and the physical topology definition. The topology is defined in the `Connections` class via a method that takes the name of the queue and returns the connection string to be used to access that queue.
+
+snippet: GetConnectionString
+
+The `StartsWith` comparison ensures that the [satellite](https://docs.particular.net/nservicebus/satellites/) queues are correctly addressed. The `poison` queue is used by the adapter for unrecoverable failures. 
```

Comment:
>used by the adapter for unrecoverable failures. 

...which begs the question of: What happens to the messages in the poison queue? Who needs to monitor that queue and what needs to be done with the messages ending up there? Maybe a link to how to a doco page if we have one on do manage this.